hack/helm/pin_runtime_chart_version.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# USAGE:
+#   FLUID_HOME=/path/to/fluid
+#   cd $FLUID_HOME
+#   bash hack/helm/pin_runtime_chart_version.sh <fluid_version>
+
+set -e
+
+fluid_version=$1
+if [ -z "$fluid_version" ]; then
```

Comment:
I suggest using '[[' instead of '[' for conditional tests. The '[[' construct is safer and more feature-rich.